### ISSUE TITLE
Adjust color of OuiToolTip

### ISSUE THREAD

**Issue by KrooshalUX** [opensearch-project/oui#953]

**Comment by KrooshalUX** [comment]
@joshuarrrr  @BSFishy  can you supply the color sass variable information that is being used for this component, and its mapping back to the base sass variables? Are there any cases where the color for tooltip is transformed? 

Is there an opportunity for us to transform the tooltip on the fly to cover various scenarios (over a vis, on the page background, on a panel background, etc)

**Comment by BSFishy** [comment]
Looks like most of the color info is set here:

https://github.com/opensearch-project/oui/blob/6e3d31ad31ef9af9ba67365b41f47773bad4dcbc/src/global_styling/mixins/_tool_tip.scss#L13-L16

If needed, these colors can be extracted into variables that can be changed independently by themes

On the fly color transform would be more difficult, as all this is compiled ahead of time, so we'd need to make variant classes for all color combos we'd want. Still not too much effort at all, but more than just taking the variables and allowing them to be independently themeable

**Comment by joshuarrrr** [comment]
Here's the definition of `$ouiTooltipBackgroundColor`:
$ouiTooltipBackgroundColor: tintOrShade($ouiColorFullShade, 25%, 100%) !default;

**Comment by joshuarrrr** [comment]
> Is there an opportunity for us to transform the tooltip on the fly to cover various scenarios (over a vis, on the page background, on a panel background, etc)

Yeah I think this would require a little more thought and technical design, as there are other scenarios that would be difficult to account for (such as tooltips on visualizations where the visualization background is set via `<canvas>`).

>  Are there any cases where the color for tooltip is transformed?

Not that I found or am aware of

**Comment by KrooshalUX** [comment]
@joshuarrrr  lets try value #293847
